sql/core/src/main/scala/org/apache/spark/sql/SQLImplicits.scala

```diff
@@ -108,6 +108,10 @@ abstract class SQLImplicits extends LowPrioritySQLImplicits {
   /** @since 2.0.0 */
   implicit def newBoxedBooleanEncoder: Encoder[java.lang.Boolean] = Encoders.BOOLEAN
 
+  /** @since 2.4.0 */
```

Comment:
@fangshil I think this will need to be updated to `2.5.0` now that 2.4.0 has been released

Reply:
It will have to be 3.0.0 as there won't be a 2.5.0